index.bs

```diff
@@ -446,6 +446,359 @@ A file containing a 'pict' track compliant with this profile is expected to list
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used when an image is derived from another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>prem</td>
+            <td>-</td>
+            <td>MIAF</td>
+            <td>&nbsp;</td>
+            <td>Used when when an alpha image contains premultiplied color values from another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>cdsc</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used to link metadata with an image</td>
+        </tr>
+        <tr>
+            <td>idat</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>meta</td>
+            <td>Used to store derived image definitions</td>
+        </tr>
+        <tr>
+            <td>grpl</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>meta</td>
+            <td>Used to indicate that multiple images are semantically grouped</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>altr</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>&nbsp;</td>
+            <td>Used when images in a group are alternative to each other</td>
+        </tr>        
+        <tr>
+            <td>pasp</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>ipco</td>
+            <td>Used to signal pixel aspect ratio. If present, shall indicate a pixel aspect ratio of 1:1</td>
+        </tr>
+        <tr>
+            <td>colr</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>ipco</td>
+            <td>Used to signal information such as color primaries.</td>
+        </tr>
+        <tr>
+            <td>auxC</td>
+            <td>&nbsp;</td>
+            <td>0</td>
+            <td>HEIF</td>
+            <td>ipco</td>
+            <td>Used to signal the type of an auxiliary image (e.g. alpha, depth).</td>
+        </tr>
+        <tr>
+            <td>clap</td>
+            <td>&nbsp;</td>
+            <td>0</td>
+            <td>ISOBMFF</td>
+            <td>ipco</td>
+            <td>Used to signal cropping applied to an image</td>
+        </tr>
+        <tr>
+            <td>irot</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>ipco</td>
+            <td>Used to signal a rotation applied to an image</td>
+        </tr>
+        <tr>
+            <td>imir</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>ipco</td>
+            <td>Used to signal a mirroring applied to an image</td>
+        </tr>
+        <tr>
+            <td>clli</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>MIAF</td>
```

Comment:
HEIF (`wxxxxx_23008-12_Ed2_FDIS+COR1_R1+FDAM1_R1` specifically) says `clli` is defined in ISOBMFF, but my version (`m47256_14496-12_2015(E)-6th+FDAM1+FDAM2+COR1+KB-R6+6thCOR1+AMD1_draft04-must`) does not have it. Either the HEIF draft is wrong or this should say ISOBMFF instead.

Reply:
(Haven't been able to check if it's a full or simple box due to not finding it)